src/views/notifications/Latest.vue

```diff
@@ -0,0 +1,324 @@
+                        <div class="font-medium text-center lg:text-left lg:mt-3">
+                          Contact Details
+                        </div>
+                        <div class="flex flex-col justify-center items-center lg:items-start mt-4">
+                          <div class="truncate sm:whitespace-normal flex items-center">
+                            <MailIcon class="w-4 h-4 mr-2"/>{{ this?.single_notification?.user.email }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <UserIcon class="w-4 h-4 mr-2"/>{{ this?.single_notification?.user?.pre_name }} {{ this?.single_notification?.user?.last_name }}
+                          </div>
+                          <div class="truncate sm:whitespace-normal flex items-center mt-3">
+                            <HashIcon class="w-4 h-4 mr-2"/>User ID: {{ this?.single_notification?.user.id }}
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="p-5">
+                    <div class="flex flex-col-reverse xl:flex-row flex-col">
+                      <div class="flex-1 mt-6 xl:mt-0">
+                        <div class="grid grid-cols-12 gap-x-5 mb-3">
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Title
+                              </label>
+                              <input
+                                id="update-title-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.title != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Title"
+                                v-model="this.single_notification.title"
+                              />
+                              <div v-if="this.validation_error?.title != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.title[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Content
+                              </label>
+                              <textarea
+                                id="update-content-notification"
+                                :class="'form-control' + (this.validation_error?.content != null ? ' border-theme-6' : '')"
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/5172bc53bad0121ee21691def6813959202839fa